### PR TITLE
Delete book

### PR DESCRIPTION
**Pull request by IrynaKamenska**

### CODE REVIEW

File: frontend/src/DeleteBook.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+    const closeModal = () => {
+        setModalIsOpen(false)
+    }
+
+    return <>
+        <button type={"submit"} onClick={openModal}>Delete</button>
+        <Modal
+            isOpen={modalIsOpen}
+            onRequestClose={closeModal}
```
Comment (ToLa09) [inline frontend/src/DeleteBook.tsx:32]: FYI: you can call the setter inline, so you don't need the extra method: ={() => setModalIsOpen(true)}

**Review by ToLa09 — APPROVED** [review]

File: frontend/src/BookOverview.tsx [hunk 2]
```diff
@@ -27,15 +25,11 @@ function BookOverview() {
...
-        return <>Bitte haben Sie einen Augenblick Geduld...</>
-    }*/
     return <>
-        <CreateBookModal modalIsOpen={modalIsOpen} closeModal={closeModal} reloadAllBooks={fetchAllBooks}/>
+        <CreateBook modalIsOpen={modalIsOpen} closeModal={closeModal} reloadAllBooks={fetchAllBooks}/>
         {books.length > 0 ?
 
             <>
-                <div>
                     <table>
```
Comment (chris-yooo) [inline frontend/src/BookOverview.tsx:33]: hm maybe you can here use https://www.w3schools.com/html/html5_semantic_elements.asp elements and style them :)

File: frontend/src/BookOverview.tsx [hunk 3]
```diff
@@ -51,21 +45,24 @@ function BookOverview() {
...
+
                             </tr>;
                         })}
                         </tbody>
                     </table>
-                </div>
                 <button type={"submit"} onClick={openModal}>New Book</button>
             </>
                 :
                 <div>
```
Comment (chris-yooo) [inline frontend/src/BookOverview.tsx:61]: here same

File: frontend/src/DeleteBook.tsx [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    const openModal = () => {
+        setModalIsOpen(true)
+    }
+
+    const closeModal = () => {
+        setModalIsOpen(false)
+    }
+
+    return <>
+        <button type={"submit"} onClick={openModal}>Delete</button>
```
Comment (chris-yooo) [inline frontend/src/DeleteBook.tsx:29]: you use here an type submit but the button isnt within an form so change the type to button :)

**Review by chris-yooo — CHANGES_REQUESTED** [review]

**Pull request merged**